manifest.yml

```diff
@@ -14,22 +14,25 @@ applications:
   command: npm start
```

Comment:
Just needs binding to the `selfservice-secret-service`

Reply:
👍